docs/move_to_poetry.md

```diff
@@ -0,0 +1,52 @@
+## Convert pyproject.toml to use poetry from flit_core
+
+The following is a guide on converting the pyproject.toml file of the Beam app to use poetry instead of flit_core. Changes made in the file to adapt it for poetry are outlined below:
+
+ - Changed `[project]` to `[tool.poetry]`
+ - Added version key to `[tool.poetry]`
+ - Removed `requires-python` and `dynamic`
+ - Changed format to specify Authors
+
+#### Changed build System
+
+| flit_core | poetry |
+| --- | --- |
+| ```[build-system]```<br>```requires = ["flit_core >=3.4,<4"]```<br>```build-backend = "flit_core.buildapi"``` | ```[build-system]```<br>```requires = ["poetry-core"]```<br>```build-backend = "poetry.core.masonry.api"```|
+
+#### To add dependency
+```
+$ poetry add --group dev {python-package}
```

Comment:
I think the dev and regular dependency sections are switched here. The group command is new to me too, but I see now that `--dev ` is deprecated.